src/fides/core/config/security_settings.py

```diff
@@ -45,7 +45,8 @@ class SecuritySettings(FidesSettings):
     encoding: str = "UTF-8"
     env: SecurityEnv = SecurityEnv.DEV
 
-    cors_origins: List[str] = []
+    cors_origins: Union[str, List[str]] = []
```

Comment:
This change allows for `-e FIDES__SECURITY__CORS_ORIGINS="..."` to be passed through as a string without triggering a JSON validation error.

Reply:
Import to note, this looks like it will allow the config var itself to resolve to a string. In reality it's always a list. e.g. with `FIDES__SECURITY__CORS_ORIGINS: "http://localhost:4000, http://localhost:4001"` inside `docker-compose.yml` the var resolves as:

```
In [1]: from fides.core.config import get_config

In [2]: get_config().security.cors_origins
Out[2]: ['http://localhost:4000', 'http://localhost:4001']
```

[due to this validation block returning `values`](https://github.com/ethyca/fides/blob/main/src/fides/core/config/security_settings.py#L84-L88).